drivers/gps/nrf9160_gps/Kconfig

```diff
@@ -38,6 +38,14 @@ config NRF9160_GPS_HANDLE_MODEM_CONFIGURATION
 	  application send the AT commands to configure the GPS
 	  properly.
 
+config NRF9160_GPS_PRIORITY_WINDOW_TIMEOUT_SEC
+	int "Enable GPS priority window if the GPS is blocked for this duration"
+	default 10
+	help
+	  If the GPS is blocked for more than this specified amount of time and
+	  GPS priority window is enabled in the passed in GPS configuration,
+	  GPS priority window is enabled by the driver.
```

Comment:
Perhaps emphasize that GPS priority window is requested, but not necessarily granted. Also, it could be added a sentence or two with the trade-offs with short timeout before requesting priority window with regards to LTE uplink availablity.

Reply:
Changed the documentation. Please give some feedback if the change is sufficient enough.